Gruntfile.js

```diff
@@ -122,7 +131,7 @@ module.exports = function(grunt) {
 	grunt.loadNpmTasks("grunt-contrib-watch");
 	grunt.loadNpmTasks("grunt-contrib-jshint");
 	grunt.loadNpmTasks("grunt-contrib-uglify");
-	grunt.loadNpmTasks("grunt-contrib-qunit");
+	grunt.loadNpmTasks("grunt-qunit-istanbul");
```

Comment:
Maybe you should consider using [load-grunt-tasks](https://github.com/sindresorhus/load-grunt-tasks) like we do in core?